helm/chaos-mesh/values.yaml

```diff
@@ -295,6 +295,13 @@ dashboard:
     # you set a database encryption secret. This must be set before any secrets are stored
     # in the database.
     # DATABASE_SECRET:
+
+    # Set the sync period to clean up archived data
+    CLEAN_SYNC_PERIOD: 12h
+    # Set TTL of archived event data
+    TTL_EVENT: 168h
```

Comment:
Could we use `7d` and `14d` instead? `168h` is not intuitive 🤔

Reply:
Yes, I will improve it in another single pr.